packages/block-editor/src/components/inserter/hooks/use-patterns-state.js

```diff
@@ -30,8 +32,18 @@ const usePatternsState = ( onInsert, rootClientId ) => {
 				__experimentalUserPatternCategories,
 				__experimentalBlockPatternCategories,
 			} = getSettings();
+			let sectionRootClientId;
+
+			if ( __unstableGetEditorMode() === 'zoom-out' ) {
+				sectionRootClientId = unlock(
```

Comment:
I wonder if we should do this earlier - in usePatternCategories, or even in InserterMenu?

Reply:
I think doing it here in usePatternsState is fine - we're already using the `blockEditorStore` here so I don't think it feels too late. I looked at doing it earlier and it felt kind of out of place in usePatternCategories, InserterMenu, and BlockPatternsTab - it seems to require more code as we're not already calling the store in these files.

Reply:
I wonder if actually we want to create a new selector for this, so we can access it in different places.

Reply:
Trying that here: https://github.com/WordPress/gutenberg/pull/61464